backend/src/main/resources/application-prod.yml

```
@@ -0,0 +1,32 @@
+server:
+  port: 5000
+
+spring:
+  jpa:
+    hibernate:
+      ddl-auto: validate
+    properties:
+      hibernate:
+        show_sql: true
```

```suggestion
properties:
        hibernate:
          show_sql: false
          format_sql: true
          use_sql_comment: false
```
이쪽도 이렇게 바꿔주셔야 로그가 정상적으로 출력됩니다!!